### PR TITLE
Dockerfile: apt update and install should be single RUN

### PR DESCRIPTION
**Pull request by pmtk**
Ref: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,8 +34,7 @@ RUN ["make", "manager"]
 
 FROM debian:bullseye-slim
 
-RUN ["apt", "update"]
-RUN ["apt", "install", "-y", "ca-certificates"]
+RUN ["bash", "-c", "apt-get update && apt-get install -y ca-certificates"]
```
Comment (qbarrand) [inline Dockerfile]: Thank you for the link. It makes sense indeed.
I think we can use the shell form here:
```suggestion
RUN apt update && apt install -y ca-certificates
```

Comment (pmtk) [reply Dockerfile]: Just in this one place or everywhere?

Comment (qbarrand) [reply Dockerfile]: I usually try to only use the shell form when there is no other way 🙂

Comment (pmtk) [reply Dockerfile]: Changed, PTAL

**Pull request merged**